docs/usage/key-concepts/presets.md

```diff
@@ -16,6 +16,20 @@ Use presets to:
 - Share your configuration with others
 - Use somebody else's configuration and extend it with your own rules
 
+## How to use presets
+
+Let's say you're using the `config:base` preset, and want to pin GitHub Action digests.
+Instead of writing your own Renovate config, you search through Renovate's built-in presets.
+You find the the `helpers:pinGitHubActionDigests` preset and add it to the `extends` array:
+
+```json
+{
+  "extends": ["config:base", "helpers:pinGitHubActionDigests"]
+}
+```
+
+Renovate now follows the rules for `config:base` plus the rules for `helpers:pinGitHubActionDigests`.
```

Comment:
What if there's a conflict between two presets in the `extends` array? Is there a tie-breaker, like the last preset in the array wins?

Reply:
last preset wins

Reply:
```suggestion
Renovate now follows the rules for `config:base` plus the rules for `helpers:pinGitHubActionDigests`.
If there is a logical conflict between presets, then the last preset in the array wins.
```